### PR TITLE
add default laravel 7 project

### PR DESCRIPTION
**Pull request by fabriceyopa**
## Description
Add Laravel 7 project,

Just to be sure that is it the way to go

### CODE REVIEW

**Review by WilChrist — CHANGES_REQUESTED** [review]
@fabriceyopa Nice Job, sounds good. But please try to always have the PR checks proposed in the default template of github. Can you add that, please?

Please also link the issue this PR is resolving.

**Comment by WilChrist** [comment]
For the next time please name your branches according to the coventions Andela suggest us here : https://docs.google.com/presentation/d/1hvnwv76gX98nAAgTxJp47CremzwPkfzDkXgejQGz2l0/edit?usp=sharing

@kouffen you can review like this...

**Comment by fabriceyopa** [comment]
Ok, let me check that

**Comment by fabriceyopa** [comment]
Please @WilChrist can you give me again the template for the pull request. And what about the next reviewer?

**Comment by WilChrist** [comment]
@fabriceyopa that is (you can hae it by doing like if you want to create a new PR):

## Description
Please include a summary of the change and relevant motivation and context. 
List any dependencies that are required for this change.

All pull requests should be related to open issues. Indicate the issue(s) bellow and remove this line:
Fixes #(issue number 1) #(issue number 2 if applicable)

## How Has This Been Tested?
Please describe in detail how you tested your changes.
Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code etc.

## Screenshots (if applicable, else remove this line / section)


## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

**Comment by WilChrist** [comment]
The next reviewer is @kouffen

File: composer.json [hunk 1]
```diff
@@ -1,23 +1,63 @@
 {
-  "name": "",
```
Comment (kouffen) [inline composer.json:2]: Here i think , we can fill the name of our app. so we can put "name : 'ImNotFoolJustSick'

File: composer.json [hunk 2]
```diff
@@ -1,23 +1,63 @@
...
-  "description": "",
```
Comment (kouffen) [inline composer.json:3]: for description , i propose this [ "description": "the backend part for the project ImNotFoolJustSick"

**Review by kouffen — COMMENTED** [review]
Nothing to Add, we can add features now. our base project is ready.

**Pull request closed without merging**